rdr_service/dao/genomics_dao.py

```diff
@@ -1447,6 +1447,26 @@ def get_feedback_equals_record_count(self):
 
         return list(results)
 
+    def get_feedback_count_within_threshold(self, theta):
+        """
+        Retrieves feedback records where feedback count is >= a threshold of record_count
+        :param theta: threshold
+        :return: list of feedback records
+        """
+        with self.session() as session:
+            results = session.query(GenomicManifestFeedback).join(
+                GenomicManifestFile,
+                GenomicManifestFile.id == GenomicManifestFeedback.inputManifestFileId
+            ).filter(
+                GenomicManifestFeedback.ignoreFlag == 0,
+                GenomicManifestFeedback.feedbackComplete == 0,
+                GenomicManifestFeedback.feedbackRecordCount != 0,
+                GenomicManifestFeedback.feedbackRecordCount >= GenomicManifestFile.recordCount * theta,
+                GenomicManifestFeedback.feedbackManifestFileId.is_(None),
+            ).all()
+
+        return list(results)
```

Comment:
Can probably remove the outer `list` declaration, `all()` returns a list.

Reply:
Thanks, I removed the redundant `list()`